cli_reference/get_started_cli.adoc

```diff
@@ -24,15 +30,20 @@ Logged into 'https://my-openshift-server.com' as 'myuser'.
 Using project 'project01'.
 Welcome to OpenShift v3! Use 'osc --help' for a list of commands available.
 ----
+====
 
```

Comment:
Please wrap this in an Example block and provide an intro sentence to the example, as per our guidelines.